notebooks/concurrent_tasks.ipynb

```diff
@@ -0,0 +1,1224 @@
+{
```

Comment:
<p>Same here, a simple explanation of the custom component itself would be handy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
done